__tests__/endpoint.test.js

```diff
@@ -81,14 +81,67 @@ describe("/api/articles/", () => {
       });
   });
 });
+describe("/api/articles/:article_id/comments", () => {
+  test("The endpoint will respond with an array of comments from the given article ID", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
As we are asserting inside a forEach, we should also make sure that the array is not empty here